client/my-sites/hosting/staging-site-card/staging-site-production-card.tsx

```diff
@@ -1,12 +1,10 @@
-import { Button, Card, Gridicon } from '@automattic/components';
+import { Button, Card } from '@automattic/components';
 import styled from '@emotion/styled';
 import { useI18n } from '@wordpress/react-i18n';
 import clsx from 'clsx';
 import { localize } from 'i18n-calypso';
 import { useEffect, useState } from 'react';
 import { connect } from 'react-redux';
-import dividerPattern from 'calypso/assets/images/hosting/divider-pattern.svg';
```

Comment:
I think this is the only place where we use this `svg` divider. I've searched for `divider-pattern.svg` in the code base and this is the only result I got.

Let's remove the file too.

Reply:
Good call. I've removed that in [104346e](https://github.com/Automattic/wp-calypso/pull/92611/commits/104346eceb08d858802b57191248602cc347d1bd)